### DISCUSSION TITLE
the bitmex-ws.py example breaks with bitmex-websocket and bitmex-ws

### DISCUSSION THREAD

**Discussion by baardev** [highfestiva/finplot#271] [Q&A]
The bitmex example doesn't seem to work.  

**bitmex-websocket** doesn't work (and has stopped being developed/managed) at all, and has been replaced by **bitmex-ws**, which is quite different in a few ways.  

with **bitmex-websocket**

```
Traceback (most recent call last):
  File "ws.py", line 5, in <module>
    from bitmex_websocket import Instrument
ImportError: cannot import name 'Instrument' from 'bitmex_websocket' (~/src/jmcap/venv/lib/python3.9/site-packages/bitmex_websocket.py)
```

with **bitmex-ws**

`ImportError: cannot import name 'Instrument' from 'bitmex_websocket' (~/src/jmcap/venv/lib/python3.9/site-packages/bitmex_websocket.py)`

replacing the original 

`ws = Instrument(channels=[InstrumentChannels.trade, InstrumentChannels.orderBook10])`

with

`ws = BitMEXWebsocket(endpoint="https://www.bitmex.com/api/v1", symbol="ETHUSD", api_key="...",api_secret="...?)`

results in data being read, but

```
    @ws.on('action')
AttributeError: 'BitMEXWebsocket' object has no attribute 'on'
```

Any chance finplot will be updated?

**Comment by highfestiva** [comment]
Absolutely, and if you would send me a nice pull request it would be even faster! ;)
